src/lib.rs

```diff
@@ -348,6 +348,304 @@ languages_table!(
     (Zu, "zu", "zul", "zul", "Zulu", "Niger–Congo"),
 );
 
+#[cfg(feature = "iterable")]
+pub mod iter {
+    use super::LanguageCode;
+
+    #[derive(Debug, Default, Clone)]
+    pub struct Iter(usize);
+
+    impl Iterator for Iter {
+        type Item = &'static LanguageCode;
+
+        fn next(&mut self) -> Option<Self::Item> {
+            if self.0 == CODES.len() {
+                return None;
+            }
+            let current = self.0;
+            self.0 += 1;
+            Some(&CODES[current])
+        }
+    }
+
+    impl Iter {
+        pub fn as_codes(self) -> Codes {
```

Comment:
Again, documentation. And because this takes ownership it should be called `into_{codes, codes_t, codes_b}`.

Reply:
Done.

One thing tho, for the previous comment, I don't think `codes` need to consume `self` since it avoid creating and consuming an instance.
basically :
```rust
impl LanguageCode {
    pub fn codes() -> Codes {
        Codes::default()
    }
}
```
should be enough, no?

Reply:
What I mean is that it's more natural and less confusing to do :
```rust
let languages = LanguageCode::codes();
```
than :
```rust
let languages = LanguageCode::En.codes();
```

Reply:
Yes, the former is much better - sorry if I wasn't clear about that.